packages/stable/src/api/geospatial/featureAPI.ts

```diff
@@ -190,4 +191,31 @@ export class FeatureAPI extends BaseResourceAPI<GeospatialFeatureResponse> {
       this.url(`${featureTypeExternalId}/features/aggregate`)
     );
   };
+
+  /**
+   * [List features](https://docs.cognite.com/api/v1/#tag/Geospatial/operation/listFeatures)
+   *
+   * ```js
+   * const searchParams = {
```

Comment:
Do not use the term `search` as it can be confused with a search endpoint.
What about `params = {...}`?

Reply:
Makes sense.